src/main/java/org/elasticsearch/index/analysis/SkroutzGreekStemmer.java

```diff
@@ -388,6 +388,8 @@ private int rule6(char s[], int len) {
           exc6.contains(s, 0, len) ||
           endsWith(s, len, "φοιν"))
         len += 2; // add back -ικ
+      else if (String.valueOf(s).startsWith("πασχαλιατ"))
```

Comment:
Why is this needed, I don't get it! Aren't all cases covered by the first or second branches of the previous conditional? (I'm obviously missing something here.)